<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Solvionyx Control Center</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="headerLabel">
      <property name="text">
       <string>Solvionyx Control Center</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="systemGroup">
      <property name="title">
       <string>System</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0"><widget class="QLabel" name="label_1"><property name="text"><string>OS</string></property></widget></item>
       <item row="0" column="1"><widget class="QLabel" name="osLabel"><property name="text"><string>Solvionyx OS</string></property></widget></item>

       <item row="1" column="0"><widget class="QLabel" name="label_2"><property name="text"><string>Version</string></property></widget></item>
       <item row="1" column="1"><widget class="QLabel" name="verLabel"><property name="text"><string>Aurora</string></property></widget></item>

       <item row="2" column="0"><widget class="QLabel" name="label_3"><property name="text"><string>Kernel</string></property></widget></item>
       <item row="2" column="1"><widget class="QLabel" name="kernelLabel"><property name="text"><string>â€”</string></property></widget></item>

       <item row="3" column="0"><widget class="QLabel" name="label_4"><property name="text"><string>Secure Boot</string></property></widget></item>
       <item row="3" column="1"><widget class="QLabel" name="secureBootLabel"><property name="text"><string>Unknown</string></property></widget></item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="actionsGroup">
      <property name="title">
       <string>Quick Actions</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0"><widget class="QPushButton" name="networkBtn"><property name="text"><string>Network</string></property></widget></item>
       <item row="0" column="1"><widget class="QPushButton" name="updatesBtn"><property name="text"><string>Updates</string></property></widget></item>
       <item row="1" column="0"><widget class="QPushButton" name="appearanceBtn"><property name="text"><string>Appearance</string></property></widget></item>
       <item row="1" column="1"><widget class="QPushButton" name="storeBtn"><property name="text"><string>Solviony Store</string></property></widget></item>
       <item row="2" column="0"><widget class="QPushButton" name="supportBtn"><property name="text"><string>Support</string></property></widget></item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="solvyGroup">
      <property name="title">
       <string>Solvy</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0"><widget class="QLabel" name="label_5"><property name="text"><string>Status</string></property></widget></item>
       <item row="0" column="1"><widget class="QLabel" name="solvyStatusLabel"><property name="text"><string>Unknown</string></property></widget></item>
       <item row="1" column="0"><widget class="QPushButton" name="solvyLaunchBtn"><property name="text"><string>Launch Solvy</string></property></widget></item>
       <item row="1" column="1"><widget class="QPushButton" name="solvyInfoBtn"><property name="text"><string>About Solvy</string></property></widget></item>
      </layout>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
